packages/fx-core/src/plugins/resource/bot/resources/strings.ts

```diff
@@ -1,22 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+import { getLocalizedString } from "../../../../common/localizeUtils";
+
 export class CommonStrings {
   public static readonly BOT_WORKING_DIR_NAME = "bot";
-  public static readonly DEFAULT_FILE_ENCODING = "utf-8";
-
-  public static readonly APP_SERVICE_PLAN = "App Service plan";
```

Comment:
Not sure where are these string used? If they are only used in our code internally and never be exposed to customer as UI element, maybe we don't need to localize them?

Reply:
those strings are used in nowhere, so I just removed them.